docs/border-color.md

```diff
@@ -116,3 +116,11 @@ For example, use `md:s-border-positive` to apply the `s-border-positive` utility
 ```html
 <div class="s-border md:s-border-positive ..."></div>
 ```
+### Arbitrary values
+If you need to use a one-off `border-{color}` value, use square brackets to generate a property on the fly using any arbitrary value. However, we recommend only using our semantic color tokens.
```

Comment:
Maybe "semantic color tokens" here should be linked to the docs for this? ...and it should probably be "semantic color **classes**", right? 😄

Reply:
Oh. Sorry. I guess it was to say to not use hexcodes or color names as arbitrary value? 😅 🙈

Reply:
...then it is correct. 😄

Reply:
ah ok! So it should stay as "semantic color tokens"? 😄